src/main/scala/com/itechart/ny_accidents/utils/ParserUtils.scala

```diff
@@ -0,0 +1,57 @@
+package com.itechart.ny_accidents.utils
+
+import java.time.LocalDateTime
+
+import com.itechart.ny_accidents.constants.GeneralConstants
+import com.itechart.ny_accidents.constants.GeneralConstants.ZERO_DATE
+import org.apache.spark.sql.Row
+import java.sql.{Date => SqlDate}
+
+import scala.util.Try
+
+object ParserUtils {
+
+  def toLongFromInt(accident: Row, column: Int): Option[Long] = {
+    Try(accident.getInt(column).toLong).filter(value => value != 0).toOption
+  }
+
+  def toLocalDate(dateStr: String, timeStr: String): LocalDateTime = {
+    val dateTimeStr = dateStr + " " + timeStr
+    DateUtils.parseDate(dateTimeStr, GeneralConstants.DATE_TIME_ACCIDENTS_FORMATTER).getOrElse(ZERO_DATE)
```

Comment:
What if for some dates you'll need to use some other date format? How would you pass it?

Reply:
I will change the date pattern, which is in constants DATE_TIME_ACCIDENTS_FORMATTER and ZERO_DATE